apps/quickstart-integrations-monday-code/README.md

```diff
@@ -0,0 +1,43 @@
+# monday code - integration quickstart
+
+A minimal example to help you get started on monday code. 
+
+> **Note:** monday code is only available for select testers. It will be available publicly in the next few months (updated: Aug 2023). 
+
+## Running the example
+
+For the example to work, you need to set it up in the monday UI, deploy the app, and set environment variables. 
+
+> Want a video demo? [Find the link here.](https://www.loom.com/share/b982f6b88fc0491f9e70fa424eb91638)
+
+### Setup
+- Download the files from Github – `npx degit github:yuhgto/welcome-apps/apps/quickstart-integrations-monday-code`
+- Create an app in monday
+- Create a “workflows” feature and use the “Node JS Quickstart” template
```

Comment:
That's a different template, right?

Reply:
No, it's the same one – built this based on the minimal NodeJS quickstart that we have. It's the one that capitalizes text using an integration